go.mod

```diff
@@ -152,3 +154,5 @@ replace k8s.io/node-api => k8s.io/node-api v0.0.0-20190918163711-2299658ad911
 replace github.com/uber-go/atomic => go.uber.org/atomic v1.5.0
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
+
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
```

Comment:
Why?

Reply:
admission-webhook build fails due to the dependency issue with `k8s.io/component-base`, refer to https://github.com/jetstack/cert-manager/issues/2432#issuecomment-562208136, after we upgrade the k8s dependencies later, this can be removed.

Reply:
[Kubernetes Dependency Upgrading Plan in TiDB Operator](https://docs.google.com/document/d/1Ya52x2E3rB0EFrFqbvIAtzxm6BR-0putdo4pzZYuIc0/edit#heading=h.35d9rb23gi2p)
FYI